service/upload/upload.go

```diff
@@ -1,12 +1,15 @@
 package uploadService
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/NyaaPantsu/nyaa/config"
 	"github.com/NyaaPantsu/nyaa/model"
 )
 
+const trackerRegex = `^(http[s]*|udp)://[a-z0-9\.:\-/?]+$`
```

Comment:
Are you sure about that? I'm not quite knowledgeable on torrents tracker url but on the list that I've I don't see the characters you provided :/ (I just googled: list of trackers url)